### PR TITLE
Animate `WhyReportPortal` block

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**
### PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, `master` for release/hotfix)?
* [x] Have you followed the guidelines in our [Dev Guide](../#readme)
* [x] Have you lint your code locally prior to submission (`npm run lint`)?
* [x] Have you run prettier (`npm run format`)?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you checked that the build output looks according to design (screen, mobile, desktop) in Figma (`npm run serve`)?

## Visuals

ezgif-4-1d7d9ce2c5

### CODE REVIEW

File: src/hooks/useMotionEnterAnimation.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  enterState,
+  transition,
+}: UseMotionEnterAnimationProps) => {
+  const shouldReduceMotion = useReducedMotion();
+
+  return ({
+    inView,
+    delay = 0,
+    additionalEffects = { hiddenAdditional: {}, enterAdditional: {} },
+  }: {
```
Comment (ViktorSoroka07) [inline src/hooks/useMotionEnterAnimation.ts]: Let's extract the type so it is easier to read the code

**Review by ViktorSoroka07 — DISMISSED** [review]

**Review by AmsterGet — DISMISSED** [review]

**Comment by ViktorSoroka07** [comment]
Please fill PR Checklist. It is there for purpose

File: src/containers/LandingPage/WhyReportPortal/WhyReportPortal.tsx [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import React, { FC } from 'react';
 import classNames from 'classnames';
+import { InView } from 'react-intersection-observer';
```
Comment (ViktorSoroka07) [inline src/containers/LandingPage/WhyReportPortal/WhyReportPortal.tsx]: You mentioned the other day that we can leverage `InView` analog from `framer-motion`. Can we?

```
const isInView = useInView(ref, { once: true })
```

from the [docs](https://www.framer.com/motion/use-in-view/###once)

Comment (yelyzavetakhokhlova) [reply src/containers/LandingPage/WhyReportPortal/WhyReportPortal.tsx]: Done

**Review by ViktorSoroka07 — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**